CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+0.16.0 / 2021-03-04
+===================
+
+* **[Feature]** Support `@` modifiers. You can learn more about it in the [Prometheus Blog](https://prometheus.io/blog/2021/02/18/introducing-the-@-modifier/)
+* **[Feature]** Support new promQL function (clamp, sgn, last_over_time)
```

Comment:
Sorry for being pedantic ;) But it's a trademark, etc...

```suggestion
* **[Feature]** Support new PromQL functions (`clamp()`, `sgn()`, `last_over_time()`)
```

Reply:
np :) that's normal.
Should be good now